moon/ros_workspace/src/aliencontrol/package.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>aliencontrol</name>
+  <version>0.1.0</version>
+  <description>Start and stop external applications from within a ROS node</description>
+  <maintainer email="aschaef@cs.uni-freiburg.de">Alexander Schaefer</maintainer>
```

Comment:
is this correct?

Reply:
the way to use the aliencontrol project is to copy the code into your project. We can integrate it differently (e.g. load from github at docker-building time or something)